N/NetCDF/bundled/patches/0001-Fix-a-number-of-OS-specific-bugs.patch

```diff
@@ -0,0 +1,675 @@
+From fcf179c6c799020871eed3ecbcc4dd34a9663c3b Mon Sep 17 00:00:00 2001
```

Comment:
These are relatively large patches, touching lots of files.  It'd be good to upstream all changes that are general and not specific to our build system, to reduce our maintenance burden.

Reply:
The large first patch is actually from upstream (@DennisHeimbigner) and merged in main but not jet available in a released version. This patch should no longer be necessary in upcoming releases. 

For the two other patches, I filled these issues/PRs:
https://github.com/Unidata/netcdf-c/issues/2303 (PR: https://github.com/Unidata/netcdf-c/pull/2304)
https://github.com/Unidata/netcdf-c/issues/2302 (PR: https://github.com/Unidata/netcdf-c/pull/2305)